ui/dashboards/src/components/PanelDrawer/PanelEditorForm.tsx

```diff
@@ -72,8 +73,13 @@ export function PanelEditorForm(props: PanelEditorFormProps) {
 
   return (
     // Grid maxHeight allows user to scroll inside Drawer to see all content
-    <form id={panelEditorFormId} onSubmit={handleSubmit}>
-      <Grid container spacing={2} sx={{ overflowY: 'scroll', maxHeight: '90vh' }}>
+    <Box
+      component="form"
```

Comment:
Nice, I like how you used `component="form"` there are a few other places where use `<form>` directly that we could update later too